src/getting-started/introduction.md

```diff
@@ -5,13 +5,13 @@ redirect_from:
 <!--{% endif %}-->
 <!--{% if "Default.EE Only" contains site.edition %}-->
-Welcome to the next generation of the world’s leading digital commerce platform! Magento provides online merchants with unparalleled flexibility and control over the look, content, and functionality of their online stores. Magento’s intuitive Admin features powerful marketing, search engine optimization, and product management tools that give you the power to create sites that are tailored to your unique business needs. Robust and scalable, Magento offers you a stable, secure, and customizable solution for your growing business.
+Welcome to the next generation of the world’s leading digital commerce platform! Magento Commerce provides online merchants with unparalleled flexibility and control over the look, content, and functionality of their online stores. Magento’s intuitive Admin features powerful marketing, search engine optimization, and product management tools that give you the power to create sites that are tailored to your unique business needs. Robust and scalable, Magento offers you a stable, secure, and customizable solution for your growing business.
 <!--{% endif %}-->
 <!--{% if "Default.B2B Only" contains site.edition %}-->
-Welcome to the business-to-business implementation of the world’s leading digital commerce platform! Magento Commerce B2B is a set of modules that extend the functionality of Magento Commerce 2.x to meet the needs of companies who sell directly to other companies, rather than to consumers. Magento combines the convenience of business to-consumer commerce with advanced account management, robust inventory control, negotiated pricing, bulk order management, and more. Magento’s Admin back office features powerful marketing, search engine optimization, and product management tools that give you the power to create sites that are tailored to your unique business needs. Robust and scalable, Magento offers you a stable, secure, and customizable solution for your growing business.
+Welcome to the business-to-business implementation of the world’s leading digital commerce platform! Magento Commerce for B2B includes a set of modules that extend the functionality of Magento Commerce to meet the needs of companies who sell directly to other companies, rather than to consumers. Magento combines the convenience of business to-consumer commerce with advanced account management, robust inventory control, negotiated pricing, bulk order management, and more. Magento’s Admin back office features powerful marketing, search engine optimization, and product management tools that give you the power to create sites that are tailored to your unique business needs. Robust and scalable, Magento offers you a stable, secure, and customizable solution for your growing business.
```

Comment:
```suggestion
Welcome to the business-to-business implementation of the world’s leading digital commerce platform! Magento Commerce for B2B includes a set of modules that extend the functionality of Magento Commerce to meet the needs of companies who sell directly to other companies, rather than to consumers. Magento combines the convenience of business-to-consumer commerce with advanced account management, robust inventory control, negotiated pricing, bulk order management, and more. Magento’s Admin back office features powerful marketing, search engine optimization, and product management tools that give you the power to create sites that are tailored to your unique business needs. Robust and scalable, Magento offers you a stable, secure, and customizable solution for your growing business.
```

Reply:
changed " business to-consumer" to " business-to-consumer"